### DISCUSSION TITLE
Does this SDK handle CRM API throttling errors?

### DISCUSSION THREAD

**Discussion by michaelmmcrae1** [microsoft/PowerPlatform-DataverseServiceClient#236] [Q&A] [answered]
Does the SDK/client handle  throttling errors on its own... or does it throw a certain exception we can look for?

In other words... if using this library, how should we plan/expect to handle CRM throttling?

**Comment by MattB-msft** [comment, marked as answer]
The library handles dataverse throttling and retriable errors internally.  You can disable this behavior by setting the client max retry property to 0
________________________________
From: Michael McRae ***@***.***>
Sent: Wednesday, February 9, 2022 3:18:25 PM
To: microsoft/PowerPlatform-DataverseServiceClient ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [microsoft/PowerPlatform-DataverseServiceClient] Does this SDK handle CRM API throttling errors? (Discussion #236)


Does the SDK/client handle throttling errors on its own... or does it throw a certain exception we can look for?

In other words... if using this library, how should we plan/expect to handle CRM throttling?

**Reply by michaelmmcrae1** [reply]
I very much appreciate your quick response.
